src/applications/caregivers/tests/unit/components/FormPages/FacilityConfirmation.unit.spec.jsx

```diff
@@ -47,65 +46,75 @@ describe('CG <FacilityConfirmation>', () => {
     goToPath.reset();
   });
 
-  context('formNavButtons', () => {
-    it('renders back and forward buttons', () => {
-      const { selectors } = subject();
-      expect(selectors().formNavButtons.back).to.exist;
-      expect(selectors().formNavButtons.forward).to.exist;
+  context('when the page renders in the normal form flow', () => {
+    it('should render the selected facility name and address when data is provied', () => {
+      const { queryByText } = subject();
+      const address = selectedFacility.address.physical;
+      expect(queryByText(new RegExp(selectedFacility.name))).to.exist;
+      expect(queryByText(new RegExp(address.address1))).to.exist;
+      expect(queryByText(new RegExp(address.address2))).to.exist;
+      expect(queryByText(new RegExp(address.address3))).to.exist;
     });
 
-    it('calls goBack callback on click', () => {
+    it('should render the assigned support facility name and address when data is provided', () => {
+      const { queryByText } = subject();
+      const address = caregiverFacility.address.physical;
+      expect(queryByText(new RegExp(caregiverFacility.name))).to.exist;
+      expect(queryByText(new RegExp(address.address1))).to.exist;
+      expect(queryByText(new RegExp(address.address2))).to.exist;
+      expect(queryByText(new RegExp(address.address3))).to.exist;
+    });
+
+    it('should render just the facility name when address is omitted from facility data', () => {
+      const { queryByText } = subject({
+        assigned: { ...caregiverFacility, address: undefined },
+      });
+      const address = caregiverFacility.address.physical;
+      expect(queryByText(new RegExp(caregiverFacility.name))).to.exist;
+      expect(queryByText(new RegExp(address.address1))).to.not.exist;
+      expect(queryByText(new RegExp(address.address2))).to.not.exist;
+      expect(queryByText(new RegExp(address.address3))).to.not.exist;
+    });
```

Comment:
Added this test to cover the case where a facility address is not in the returned data.